polaris-react/.storybook/preview.js

```diff
@@ -6,6 +6,17 @@ import {GridOverlay} from './GridOverlay';
 import {RenderPerformanceProfiler} from './RenderPerformanceProfiler';
 import {gridOptions, featureFlagOptions} from './manager';
 import {breakpoints} from '@shopify/polaris-tokens';
+import isChromatic from 'chromatic/isChromatic';
+
+// Use the document.fonts API to ensure fonts have fully loaded before rendering a story. Important for consistency when doing visual diffing in Chromatic.
+// See: https://developer.mozilla.org/en-US/docs/Web/API/Document/fonts
+const fontLoader = async () => ({
+  fonts: await document.fonts.ready,
+});
+
+// Exporting as a loader from `preview.js` forces the promise to be resolved before EVERY story
```

Comment:
I tried removing it but it still flashes the `SF` font locally...
is there any way we can test that it isn't flashing on chromatic?

It might be worth merging both this approach AND https://github.com/Shopify/polaris/pull/10073?

<img width="1288" alt="Screenshot 2023-08-16 at 10 21 13 AM" src="https://github.com/Shopify/polaris/assets/20652326/4dc03750-bcd3-4711-b984-cc26a842a851">